lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -946,11 +946,7 @@ def set_proj_type(self, proj_type):
     def get_proj(self):
         """
         Create the projection matrix from the current viewing position.
```

Comment:
If the docstring is just a single line, we use the more compact notation:

~~~
def _get_proj(self):
    """Create the projection matrix from the current viewing position."""
~~~

Reply:
Thank you for your patience. I updated the docstring to follow the specified notation; I also updated another method's one line docstring in the same file which wasn't following the notation. There is a docstrings I am not sure about, as it seems incomplete:
```    
   def _3d_extend_contour(self, cset, stride=5):
        """
        Extend a contour in 3D by creating
        """
```

Reply:
Indeed that does seem to trail off mid-sentence.  You can either push more commits to this PR to try and finish the sentence or do that in a second PR.

Reply:
I can have a look at it later and open a second PR :smile: